scrapers/CockyBoys.yml

```diff
@@ -0,0 +1,43 @@
+name: "CockyBoys"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - cockyboys.com/scenes/
+    scraper: sceneScraper
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - cockyboys.com/models/
+    scraper: performerScraper
+
+xPathScrapers:
+  sceneScraper:
+    common:
+      $performer: //div[@class="movieModels"]/span/a[@class="name gothamy"]
+    scene:
+      Title: //meta[@property="og:title"]/@content
+      Date:
+        selector: //strong[contains(text(), "Released:")]/following-sibling::text()
+        postProcess:
+          - parseDate: 01/02/2006
+      Details:
+        selector: //div[@class="movieDesc"]/h2/following-sibling::text()
+        concat: "\n\n"
```

Comment:
This will need some more tweaking
Any scene that has performer links in the description (eg `https://cockyboys.com/scenes/olivier-robert-raphael-louis.html?type=vids` ) has an issue with this
From a quick look this
```yaml
      Details:
        selector: //div[@class="movieDesc"]
        postProcess:
          - replace:
              - regex: ^Description\s+
                with:
```
gets the whole description just not that well formatted :-(
Not sure if there is an easy way to get all the description with the formatting as close as possible to the original